sage-ios/SAGE/SAGE/School.swift

```diff
@@ -57,25 +57,26 @@ class School: NSObject, NSCoding {
             default: break
             }
         }
+        super.init()
```

Comment:
I made self.id not an optional property, so that needs to be assigned before calling super.init, and it felt neater to put it at the bottom than halfway through the function